lib/src/model/package.dart

```diff
@@ -118,29 +118,32 @@ class Package extends LibraryContainer
     return _documentationAsHtml;
   }
 
+  String /*?*/ _documentation;
+
   @override
   String get documentation {
-    return hasDocumentationFile
-        ? packageGraph.resourceProvider
-            .readAsMalformedAllowedStringSync(documentationFile)
-        : null;
+    if (_documentation != null) {
```

Comment:
The usual code pattern in dartdoc for this is (at least until null safety makes its way to us):

```
  if (_documentation != null) {
     // things and such
     _documentation = calculatedThing
  }
  return _documentation;
```

This makes the code more readable when browsing a lot of these caches.

Reply:
corrected typo, above